tests/ovm_test.go

```diff
@@ -237,6 +237,65 @@ var _ = Describe("OfflineVirtualMachine", func() {
 			}, 120*time.Second, 1*time.Second).Should(BeTrue())
 		})
 
+		It("should recreate VM if the VM's pod gets deleted", func() {
+			var firstVM *v1.VirtualMachine
+			var curVM *v1.VirtualMachine
+			var err error
+
+			By("Creating a new OVM")
+			newOVM := newOfflineVirtualMachine(true)
+
+			// wait for a running VM.
+			By("Waiting for the OVM's VM to start")
+			Eventually(func() bool {
+				firstVM, err = virtClient.VM(newOVM.Namespace).Get(newOVM.Name, v12.GetOptions{})
+				if err != nil {
+					return false
+				}
+				if !firstVM.IsRunning() {
+					return false
+				}
+				return true
+			}, 120*time.Second, 1*time.Second).Should(BeTrue())
+
+			// get the pod backing the VM
```

Comment:
Those `By` statements can be really helpful! This line might be able to use one too.